docs/tutorials/0_tutorial_qiskit-braket-provider_overview.ipynb

```diff
@@ -885,17 +933,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 27,
    "id": "1ec1b936",
    "metadata": {
     "slideshow": {
      "slide_type": "slide"
     }
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{}"
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "# print(job.result())\n",
-    "AwsQuantumJob(\"<JOB_ARN>\").result()"
+    "AwsQuantumJob(\"arn:aws:braket:us-west-2:649749888193:job/qiskit-braket-vqe\").result()"
```

Comment:
Please replace this back with `<JOB_ARN>` since this specific job arn belongs to you.

Reply:
this has now been completed